tests/x11/ghostscript.pm

```diff
@@ -21,20 +21,9 @@ use testapi;
 use utils;
 
 sub run() {
-    select_console "x11";
-
-    x11_start_program "xterm";
-
-    # become root, disable packagekit and install all needed packages
-    become_root;
-    pkcon_quit;
-    zypper_call "in ghostscript ghostscript-x11";
-
-    # special case for gv which is not installed on all flavors
-    my $gv_missing = zypper_call("in gv", exitcode => [0, 104]);
-
-    # exit root shell
-    type_string "exit\n";
+    select_console 'x11';
+    ensure_installed 'ghostscript ghostscript-x11 gv';
```

Comment:
@rwmanos can you give me an example product where `gv` would not be available so that I could experience that myself?